### DISCUSSION TITLE
field is not required still checking other validations

### DISCUSSION THREAD

**Discussion by Junaid300** [react-hook-form/react-hook-form#11383] [General]
I have added rules inside controller.
```
 rules={{
              required: {
                value: false,
                message: 'requred',
              },
{
        min: {
          value: 20,
          message: 'Small',
        }
      }
```
Now why it is checking for min value even field is not required. Actually, field gonna be conditional on base of switch. 
It shouldn't check for min and max value . If field is not required
Any suggestion about it ?

**Comment by Moshyfawn** [comment]
It depends on the type of input and how you handle it. A minimal reproduction codesandbox would help.
